<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Elliot</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/Image/OnePiece.png</normaloff>:/Image/OnePiece.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>81</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>性别</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="rbMan">
       <property name="text">
        <string>男</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbWoman">
       <property name="text">
        <string>女</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>81</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>婚姻</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="rbMarried">
       <property name="text">
        <string>已婚</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbUnmarried">
       <property name="text">
        <string>未婚</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>91</width>
      <height>114</height>
     </rect>
    </property>
    <property name="title">
     <string>问卷调查</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QCheckBox" name="cbHaochi">
       <property name="text">
        <string>好吃</string>
       </property>
       <property name="tristate">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbHaokan">
       <property name="text">
        <string>好看</string>
       </property>
       <property name="tristate">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbHaowan">
       <property name="text">
        <string>好玩</string>
       </property>
       <property name="tristate">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbWuliao">
       <property name="text">
        <string>无聊</string>
       </property>
       <property name="tristate">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>401</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
